lib/popcorn/pop_validation.nit

```diff
@@ -516,6 +516,45 @@ class FloatField
 	end
 end
 
+# Check if a field is a Bool
+#
+# ~~~
+# var validator = new ObjectValidator
+# validator.add new BoolField("field", required=false)
+# assert validator.validate("""{}""")
+# assert validator.validate("""{ "field": true }""")
+# assert validator.validate("""{ "field": false }""")
+# assert not validator.validate("""{ "field": "foo" }""")
```

Comment:
maybe document that there is no type juggling.

~~~nit
assert not validator.validate("""{ "field": "true" }""")
assert not validator.validate("""{ "field": 1 }""")
assert not validator.validate("""{ "field": [true] }""")
~~~